mantle/kola/tests/ostree/unlock.go

```diff
@@ -39,6 +40,17 @@ func init() {
 		Name:        "ostree.hotfix",
 		FailFast:    true,
 		Tags:        []string{"ostree"},
+		UserData:    conf.Butane(`
+variant: fcos
+version: 1.4.0
+systemd:
+  units:
+  - name: rpm-ostreed.service
+    dropins:
+    - name: 10-debug.conf
+      contents: |-
+        [Service]
+        Environment=G_MESSAGES_DEBUG=all`),
```

Comment:
should we add a comment here about removing once the issue is fixed? or is this generally useful?